zwave_js_server/model/node/__init__.py

```diff
@@ -126,134 +127,145 @@ def __eq__(self, other: object) -> bool:
             self.client.driver == other.client.driver and self.node_id == other.node_id
         )
 
+    @property
+    def data(self) -> NodeDataType:
```

Comment:
What currently needs this that we know? If we're making a breaking change anyway, maybe we don't need this?

Reply:
Yes, sounds good.